pytorch_lightning/trainer/training_loop.py

```diff
@@ -432,12 +432,13 @@ def _track_gradient_norm(self):
                 grad_norm_dict = grad_norm(model, self.trainer.track_grad_norm)
         return grad_norm_dict
 
-    def tbptt_split_batch(self, batch):
+    def _tbptt_split_batch(self, batch) -> List[Any]:
         splits = [batch]
-        if self.trainer.truncated_bptt_steps is not None:
+        truncated_bptt_enabled = self._truncated_bptt_enabled()
+        if truncated_bptt_enabled:
```

Comment:
micro performance nits: this is a check we could check do once at the start of training instead of once per batch